src/pages/VestingClaim.tsx

```diff
@@ -53,11 +53,28 @@ function VestingClaim(): ReactElement {
         const currentBeneficiaryEvents = events.filter(
           (event) => event.args?.beneficiary === account,
         )
-        const isValidBeneficiary = currentBeneficiaryEvents.length > 0
+        const nonCapturedBeneficiaryAddresses: {
+          [beneficiaryAddress: string]: string
+        } = {
+          "0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE":
+            "0x5DFbCeea7A5F6556356C7A66d2A43332755D68A5",
+          "0xEC6f7607cD7E4C942a75d40C269deC01BBc9A15e":
+            "0x597e475a5ddd90b3eb2135ac47319bd866f685d8",
+          "0x53ab66E5bAb196CF86F65feD79981cb85470200e":
+            "0x5c85B43468da23F86016f508f14cA927bfD8A737",
+          "0x17f61d0F9701A7fB5814C2d4AD3dC3831e07b277":
+            "0x41092b4ecf2c4db719ec5ab67dbd0c66f095ee97",
+        }
+        const isValidBeneficiary =
```

Comment:
you're a valid beneficiary if you're picked up in `currenctBeneficiaryEvents` or have a value in `nonCapturedBeneficiaryAddresses`